src/core/blueprint/index.rs

```diff
@@ -66,6 +66,37 @@ impl Index {
     pub fn get_mutation(&self) -> Option<&str> {
         self.schema.mutation.as_deref()
     }
+
+    pub fn is_type_implements(&self, type_name: &str, implements: &str) -> bool {
```

Comment:
```suggestion
    pub fn is_type_implements(&self, implements: &str, type_name: &str) -> bool {
```

Nit Pick: In general its a good idea to keep the most changing arguments on the right and the least changing arguments on the left. For eg: `self` doesn't change and stays on the left, similarly we are looking for `implements` in a chain of types, so keeping that as the second parameter seems intuitive.

Reply:
but it also should correspond to the name of the method otherwise it would be confusing.
Also after removal of recursive call both parameters are changing in the same magnitude so I left it unchanged